### PR TITLE
Added some block mappings and fixed a few (important) bugs with the yarn script

### PR DESCRIPTION
**Pull request by BleachDrinker420**

### CODE REVIEW

**Comment by BoogieMonster1O1** [comment]
conflicts :crab:

File: mappings/net/minecraft/block/AbstractPressurePlateBlock.mapping [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+	FIELD field_17736 DEFAULT_SHAPE Lnet/minecraft/class_4100;
+	METHOD method_11548 updatePlateState (Lnet/minecraft/class_1150;Lnet/minecraft/class_2552;Lnet/minecraft/class_2232;I)V
+		ARG 1 world
+		ARG 2 pos
+		ARG 3 state
+		ARG 4 output
+	METHOD method_11549 playPressSound (Lnet/minecraft/class_3598;Lnet/minecraft/class_2552;)V
+		ARG 1 world
+		ARG 2 pos
+	METHOD method_11550 playDepressSound (Lnet/minecraft/class_3598;Lnet/minecraft/class_2552;)V
```
Comment (BoogieMonster1O1) [inline mappings/net/minecraft/block/AbstractPressurePlateBlock.mapping:13]: unpress

**Comment by BoogieMonster1O1** [comment]
I'll do a detailed review in a bit

File: mappings/net/minecraft/block/AbstractPressurePlateBlock.mapping (hunk 1 above)
Comment (BleachDrinker420) [reply mappings/net/minecraft/block/AbstractPressurePlateBlock.mapping:13]: blame modmuss, like 99% of the mappings were copy pasted from upstream

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/block/AbstractPressurePlateBlock.mapping:13]: 👍 fine then

**Comment by BleachDrinker420** [comment]
Ready to be merged

**Comment by BleachDrinker420** [comment]
Any reviewers?

**Review by arthurbambou — APPROVED** [review]

**Review by halotroop2288 — APPROVED** [review]

**Pull request merged**